frontendUiTable/src/main/java/com/virtuslab/gitmachete/frontend/ui/table/GitMacheteGraphTableManager.java

```diff
@@ -60,6 +64,14 @@ public GitMacheteGraphTableManager(Project project,
     subscribeToGitRepositoryChanges();
   }
 
+  @SneakyThrows
+  private static IGitMacheteRepositoryFactory getGitMacheteRepositoryFactoryInstance() {
+    Reflections reflections = new Reflections("com.virtuslab");
```

Comment:
Big picture: we now require an `IGitMacheteRepositoryFactory` implementation to be on the **runtime classpath**, but not on the **compilation classpath** of `frontendUiTable`.

Reply:
Btw that's in essence the same trick that SLF4J is built upon.

Reply:
:rocket: :exclamation: